templates/workflow/version.tpl

```diff
@@ -0,0 +1,54 @@
+{**
+ * lib/pkp/templates/workflow/version.tpl
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Version view for workflow stage production
+ *}
+{if $isPublished}
+	<div class="pkp_notification">
+		<div class="notifyInfo">
+			This version has not been published yet.
+		</div>
+	</div>
+{/if}
+<div class="pkp_panel_wrapper">
+
+	<div class="pkp_context_sidebar">
+
+		{** edit metadata of this version **}
+		<div id="editMetadataDiv" class="pkp_tab_actions">
+			<ul class="pkp_workflow_decisions">
+				<li>{include file="linkAction/linkAction.tpl" action=$editMetadataLinkAction}</li>
+			</ul>
+		</div>
+		
+		{** schedule for publication **}
```

Comment:
The production.tpl is completly in OJS, maybe it would be ok to leave version.tpl there as well? Also I do not really know how to reasonably separate the OJS and pkp-lib parts, because they are mingled. 😕